platform/ui/src/components/quickSwitch/StudyItem.js

```diff
@@ -26,12 +31,7 @@ export class StudiesItem extends Component {
       >
         <div className="studyItemBox">
           <div className="studyModality">
-            <div
-              className="studyModalityText"
-              style={this.getModalitiesStyle()}
-            >
-              {modalities}
-            </div>
+            <div className="studyModalityText">{modalities}</div>
```

Comment:
When clicking on an opened study box, it would be nice if it closes (toggle opened/closed).

![Feb-20-2020 20-27-46](https://user-images.githubusercontent.com/1905961/74989633-c2f86700-541f-11ea-96a8-3c4a4a4fffad.gif)

Reply:
I will check that with Danny, I thought about that, but then testing legacy implementation it was not collapsing, so I kept this way.

Reply:
I've talked with Danny and we agreed to add expand/colapse instead of just expand. 
Code is updated.